tests/e2e/tests/checkout/checkout-block.shopper.block_theme.side_effects.spec.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { expect, test as base } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+import { REGULAR_PRICED_PRODUCT_NAME } from './constants';
+import { CheckoutPage } from './checkout.page';
+
+const test = base.extend< { checkoutPageObject: CheckoutPage } >( {
+	checkoutPageObject: async ( { page }, use ) => {
+		const pageObject = new CheckoutPage( {
+			page,
+		} );
+		await use( pageObject );
+	},
+} );
+
+test.describe( 'Shopper → Account', () => {
+	// Become a logged out user.
+	test.use( {
+		storageState: {
+			origins: [],
+			cookies: [],
+		},
+	} );
+
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.rest( {
+			method: 'PUT',
+			path: 'wc/v3/settings/account/woocommerce_enable_guest_checkout',
+			data: { value: 'yes' },
+		} );
+		await requestUtils.rest( {
+			method: 'PUT',
+			path: 'wc/v3/settings/account/woocommerce_enable_checkout_login_reminder',
+			data: { value: 'yes' },
+		} );
```

Comment:
Do you think we should move path strings to constants for code readability?

```ts
const GUEST_CHECKOUT_SETTING = 'wc/v3/settings/account/woocommerce_enable_guest_checkout';
const LOGIN_REMINDER_SETTING = 'wc/v3/settings/account/woocommerce_enable_checkout_login_reminder';
const SIGNUP_LOGIN_CHECKOUT_SETTING = 'wc/v3/settings/account/woocommerce_enable_signup_and_login_from_checkout';
```

I have a suggestion. Since we are utilizing the `await requestUtils.rest(` function to configure various settings, do you think it would be better to move it to a separate file so that it can be used easily across multiple test files? We can call that function like this.
```ts
	await configureSettings(requestUtils, GUEST_CHECKOUT_SETTING, 'yes');
	await configureSettings(requestUtils, LOGIN_REMINDER_SETTING, 'yes');
	await configureSettings(requestUtils, SIGNUP_LOGIN_CHECKOUT_SETTING, 'yes');

```